templates/goal_comments.hbs

```diff
@@ -0,0 +1,36 @@
+{{#if comments}}
+
+<details>
+<summary>{{num_comments}} update(s) posted.</summary>
+
+<!-- this comment helps to convince the markdown parser to do the right thing -->
+
+{{#each comments}}
+
+<!-- this comment helps to convince the markdown parser to do the right thing -->
+
+<a href="{{url}}">Comment by {{author}} posted on {{created_at}}:</a><br><br>
```

Comment:
![image](https://github.com/user-attachments/assets/e6402929-da46-471c-83d9-b2874e968556)

a single empty line will look better

```suggestion
<a href="{{url}}">Comment by {{author}} posted on {{created_at}}:</a><br>
```

![image](https://github.com/user-attachments/assets/41687e53-802c-4edf-9a71-ab3d7c5cbf5e)

Reply:
I'll also take care of improving the datetime format tomorrow.

Reply:
yeah I was pretty lazy

Reply:
Both of these are fixed in https://github.com/rust-lang/rust-project-goals/pull/300